Blog - How the startup data stack evolves, what we've found

ivanagas opened the pull request:
## Changes

Blog on how the data stack evolves based on [Luke's research](https://posthog.slack.com/archives/C01RJ7T212S/p1673955282524229) into the subject.

Would appreciate feedback on:
- Making the intro snappier
- Improving graphics
- Improving the "The series B and beyond data stack"
- How we want to position our vision of the future 

## Checklist
- [x] Titles are in [sentence case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case)
- [x] Feature names are in **[sentence case too]([https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case))**
- [x] Words are spelled using American English
- [x] I have checked out our [style guide](https://github.com/PostHog/posthog.com/blob/master/STYLEGUIDE.md)

charlescook-ph commented:
Just sharing big picture feedback at this stage:

- The main thing I'm struggling with is the hook here - what is the main thrust of this article? Is it a 'we interviewed hundreds of companies and this is what we've seen works best at each stage'? Which was Luke's original research (kind of). Or do we have an opinion about what we think the stack _should_ be at each company's stage, and why? Feels like the former is probably more credible, but that's not how I read the article.
  - At the moment this is all mildly interesting but it reads a bit like one of those white papers that talks about 'the state of data warehouses in 2023' (sorry!)
- It seems like you might be extrapolating a bit too much from 5 interviews to generalizations - e.g. hiring a Head of Data at Series A stage seems quite unusual? (ok I've only worked at 3 startups, but none of them hired a Head of Data at that stage, and PostHog doesn't have one)

How about if you went in a super opinionated, anti-whitepapery direction - e.g. 'we talked to _actual_ startups and this is what we learned, don't just read some 50 page PDF that's trying to sell Snowflake to your 3 person pre-seed startup'.?

ivanagas commented:
> How about if you went in a super opinionated, anti-whitepapery direction - e.g. 'we talked to _actual_ startups and this is what we learned, don't just read some 50 page PDF that's trying to sell Snowflake to your 3 person pre-seed startup'.?

The reason I didn't choose to go this way is that it would just read like a long ad for PostHog. What do we think is the best startup data stack for MVP, seed, and series A? PostHog. What happens when you hit series B+, you can connect PostHog to your "modern data stack."

I'll try working on the hook some more.

ivanagas commented:
Rewrote the intro to try to make it clearer what the goal of this piece is. 

Also rewrote the series A section to extrapolate less (specifically, hedge the "head of data" sentence).

charlescook-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,117 @@
+ - Explainers
+---
+
+The goal of the data is to help startups build a better product. To help them with this, startups use all sorts of tools, known as the startup data stack. 
```
```suggestion
The goal of data is to help startups build a better product. To help them with this, startups use all sorts of tools, known as the startup data stack. 
```

charlescook-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,117 @@
+The goal of the data is to help startups build a better product. To help them with this, startups use all sorts of tools, known as the startup data stack. 
+
+To help you understand the startup data stack, and be better able leverage it, this post will go over:
+- What the startup data stack looks like from MVP to series B and beyond
```
```suggestion
- What the startup data stack looks like from MVP to Series B and beyond
```

charlescook-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,117 @@
+
+Series A is where data begins to get serious, and by serious, I mean the startup makes someone responsible for owning the data stack. This person is often a head of data, data engineer, or backend engineer focused on data. They come in and implements the "modern data stack." This contains four tools:
+
+1. A data warehouse stores all the data and act as a single source of truth. Options include Snowflake, BigQuery, and RedShift.
```
(Obv opinionated - feel free to disagree!)

ivanagas replied:
The problem is that ClickHouse isn't really a warehouse. We are trying to make it a warehouse, but it isn't a warehouse by default.

charlescook-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,117 @@
+
+Startups need this because managing and accessing individual sources of data becomes unsustainable. Teams need a single source of truth, and the modern data stack gives them that. This creates trust in the data and greater accuracy.
+
+The downside of this stage is that engineers start to get further away from the data important to them. They must learn the modern data stack tools or rely on the data team for insights. The analytics tools set up earlier become inaccurate, and are used less. This becomes a much larger issue in the next stage.
```
Another downside is integrating multiple tools together?

charlescook-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,117 @@
+
+The downside of this stage is that engineers start to get further away from the data important to them. They must learn the modern data stack tools or rely on the data team for insights. The analytics tools set up earlier become inaccurate, and are used less. This becomes a much larger issue in the next stage.
+
+## The series B and beyond data stack
```
```suggestion
## The Series B and beyond data stack
```

charlescook-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,117 @@
+
+Tools set up earlier, such as product analytics or the CDP, are locked in and forgotten. These tools now only send data which eventually makes its way into the warehouse. If budget becomes a problem, these tools are likely the first ones to get cut.
+
+New tools, such as machine learning, are introduced to gain insights from "big data." The data team also grows to fulfill the needs of other teams for data. This does add maintenance, cost, and complexity. A larger data team is needed to manage the complexity at this scale.
```
Any specific product examples to give here? It was nice that you included this in previous sections.

ivanagas replied:
I've deleted this section now

charlescook-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,117 @@
+
+![PostHog CDP](../images/blog/startup-data-stack/posthog-cdp.png)
+
+We are also working on figuring out what PostHog would look like as [a data warehouse](https://github.com/PostHog/posthog/issues/14406). Data accuracy and reporting flexibility matter at scale. Being a warehouse enables PostHog to provide this to startups, while still using the tools PostHog offers. With PostHog as a warehouse, startups don’t need to migrate or set up many of the modern data stac ...
```
```suggestion
We are also figuring out what PostHog would look like as [a data warehouse](https://github.com/PostHog/posthog/issues/14406). Data accuracy and reporting flexibility matter at scale. Being a warehouse enables PostHog to provide this to startups, while still using the tools PostHog offers. With PostHog as a warehouse, startups don’t need to migrate or set up many of the modern data stack tools.
```

charlescook-ph replied:
Mention ClickHouse here?

charlescook-ph commented:
Yeah this makes a bunch more sense now! I think maybe the framing at the beginning was throwing me off at first. 

Have submitted a bunch of line edit suggestions - do you think it's worth doing a turn on the images to tighten them up a bit? (ie. make the logos a bit more aligned to a grid?)

andyvan-ph reviewed (commented):
My feeling is similar Charles'. This would work better as a more opinionated / "our vision" take on the "Modern Data Stack". We have all the info and I like the graphics, but it feels like information with no utility at the moment.

Walking things back, I think we start from a viewpoint of "We need to talk about The Modern Data Stack" and make it about:

1. Why it's way too complicated and inefficient for most companies
2. Gets in the way of innovation, rather enabling it
3. Our vision for how we want to fix that

The last bit may be a bit self-serving, but better to say something controversial than nothing at all.

Probably worth getting Luke and James' take on this as they've thought about it more than I!

ivanagas commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,117 @@
+- Data security, safety, and privacy requirements.
+- Lack of knowledge of data available or how to use it.
+
+As data sizes continue to grow, there are growing pains. For example, team members might need to write optimized SQL to access specific data they want. This is because everything revolves around the data warehouse. Everything adds, accesses, or transforms data from the warehouse.  The costs of hosting the warehouse as well as querying and processing data from it can become significant.
```
I've reworked this section, but I think it should have been just "data", data sizes is a subset of data that is growing.

andyvan-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,108 @@
+- How do we optimize our [conversion funnel](/tutorials/funnels)?
+- What channels should we [be advertising on](/blog/dev-marketing-for-startups)?
+
+To answer these questions, teams pull data from a growing number of sources like product analytics, CRM, help desk, and ad platform. They also utilize more of the features of these sources, such as customer engagement and session recordings, generating more data.
```
```suggestion
To answer these questions, teams pull data from a growing number of sources like product analytics, CRM, help desk, and ad platforms. They also utilize more of the features of these sources, such as customer engagement and session recordings, generating more data.
```

andyvan-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,108 @@
+
+## The Series A data stack
+
+Series A is where data begins to get serious, and by serious, I mean the startup makes someone responsible for owning the data stack. This person is often a head of data, data engineer, or backend engineer focused on data. They see their goal as empowering engineers with data to evaluate the success of what they build. To do this, they implement the "modern data stack" which contains four tools:
```
```suggestion
Series A is where data begins to get serious, and by serious, I mean the startup makes someone responsible for owning the data stack. This person is often a head of data, data engineer, or backend engineer focused on data. They see their goal as empowering engineers with data to evaluate the success of what they build. To do this, they implement the so-called "modern data stack" which contains four tools:
```

A bit of optional sass here.

andyvan-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,108 @@
+- Data security, safety, and privacy requirements.
+- Lack of knowledge of data available or how to use it.
+
+For example, engineers might need to write optimized SQL queries to understand customer feature usage. This requires an understanding of SQL, data availability, data structured, as well as access. There all create friction on what engineers want to do: build better products. 
```
```suggestion
For example, engineers might need to write optimized SQL queries to understand customer feature usage. This requires an understanding of SQL, data availability, data structured, as well as access. These all create friction on what engineers want to do: build better products. 
```

andyvan-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,108 @@
+
+We are also figuring out what PostHog looks like as [a data warehouse](https://github.com/PostHog/posthog/issues/14406) utilizing [ClickHouse](/docs/how-posthog-works/clickhouse). Data accuracy and reporting flexibility matter at scale. Being a warehouse enables PostHog to provide this to startups, and integrate it with tools they are already using PostHog for. With PostHog as a warehouse, startu ...
+
+PostHog wants to make it quick to evaluate whether products are working or not. The ability for engineers to find and analyze the metrics they want, along with tools like experiments and session recordings, is key to this. The modern data stack creates complexity and silos that make figuring out what to work on and what’s working difficult.
```
```suggestion
We want to make it easy to evaluate whether products are working or not. The ability for engineers to find and analyze the metrics they want, along with tools like experiments and session recordings, is key to this. The modern data stack creates complexity and silos that make figuring out what to work on and what’s working difficult.
```

andyvan-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,108 @@
+
+For example, engineers might need to write optimized SQL queries to understand customer feature usage. This requires an understanding of SQL, data availability, data structured, as well as access. There all create friction on what engineers want to do: build better products. 
+
+## PostHog’s vision for the future of the startup data stack
+
+Being a part of this evolution, we have thoughts on where PostHog fits into this. [Our mission](/handbook/strategy/overview) is to increase the number of successful products in the world, and we do this by helping engineers be better at product. This means helping them gather and analyze data at the earliest stage of their product, and integrating more of the parts of the startup data stack as th ...
```
I think we should frame this opening a bit more aggressively – i.e. make it "The problem with the 'modern data stack' and how we want to fix it" and open with a more opinionated take why the current norms are bad. All the right bits of information is here, it just feels a bit polite. I think we should be confident enough to say "this sucks" and here's a better way.

ivanagas replied:
got it I think

andyvan-ph reviewed (commented):
This is almost there. As per my inline comment, I think we should punch up the final section and be a bit more opinionated on why the "modern data stack" is kind of horrible for startups.

andyvan-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,108 @@
+
+The goal of the stack at this stage is to give insights into how the product and specific features are doing. Teams want to answer questions about the product and get basic KPI reporting. This enables them to make decisions about what to work on and measure progress towards goals.
+
+Moving from individual sources to a CDP makes gathering data faster, but starts to create a web of integrations and mainteinance. It is a challenge to know if data is accurate, luckily at this stage, it doesn't need to be. The Seed stage strikes the balance of giving engineers enough data to know what to prioritize with having plenty of time to build it.
```
```suggestion
Moving from individual sources to a CDP makes gathering data faster, but it's also where the rot begins as complexity and maintenance needs increase. Data accuracy problems proliferate at this stage, though for now this isn't a critical issue – engineers just need enough data to know what to prioritize.
```

andyvan-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,108 @@
+- How do we optimize our [conversion funnel](/tutorials/funnels)?
+- What channels should we [be advertising on](/blog/dev-marketing-for-startups)?
+
+To answer these questions, teams pull data from a growing number of sources like product analytics, CRM, help desk, and ad platforms. They also utilize more of the features of these sources, such as customer engagement and session recordings, generating more data.
```
```suggestion
You'll need to pull data pull data from a growing number of sources to answer these questions – product analytics, CRM, help desk, ad platform, etc. You'll also utilize more of the features of these sources, such as customer engagement and session recordings, generating more data than before.
```

andyvan-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,108 @@
+
+To answer these questions, teams pull data from a growing number of sources like product analytics, CRM, help desk, and ad platforms. They also utilize more of the features of these sources, such as customer engagement and session recordings, generating more data.
+
+Bringing all this data together and managing it often requires a [customer data platform](/docs/integrate/cdp) (CDP). CDPs collect the data from different sources and send it for use in destinations. For example, it might collect product data from PostHog, advertising data from Google, and revenue data from Stripe, and send it back to those same tools, a warehouse, or a business intelligence tool ...
```
```suggestion
This is where [customer data platforms](/docs/integrate/cdp) (CDP) like Segment come in. CDPs make it easier to bring data together by collecting it from different sources and sending it to various destinations. For example, a CDP might collect product data from PostHog, advertising data from Google, and revenue data from Stripe, and send it back to those same tools, a warehouse, or a business intelligence tool like Hex.
```

andyvan-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,108 @@
+
+## The Series A data stack
+
+Series A is where data begins to get serious, and by serious, I mean the startup makes someone responsible for owning the data stack. This person is often a head of data, data engineer, or backend engineer focused on data. They see their goal as empowering engineers with data to evaluate the success of what they build. To do this, they implement the so-called "modern data stack" which contains fo ...
```
```suggestion
Series A is where data begins to get serious. By serious, I mean (1) you need someone who is responsible for the data stack – normally a head of data, data engineer, or a backend engineer – and (2) accuracy is important.

Whomever gets the thankless task will see their goal as empowering engineers with data to evaluate the success of what they build. To do this, they implement the so-called "modern data stack" which contains four tools:
```

andyvan-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,108 @@
+
+Series A is where data begins to get serious, and by serious, I mean the startup makes someone responsible for owning the data stack. This person is often a head of data, data engineer, or backend engineer focused on data. They see their goal as empowering engineers with data to evaluate the success of what they build. To do this, they implement the so-called "modern data stack" which contains fo ...
+
+1. A data warehouse to store all the data and act as a single source of truth. Options include Snowflake, BigQuery, and RedShift.
+2. An ETL pipeline to extract data from different sources, transform it, and load it into the warehouse. Options include Fivetran, Integrate.io, and Airbyte.
+3. A data transformation tool to model data, clean it up, and make it usable. dbt is basically the only option.
+4. A business intelligence or visualization tool to get insights from the data you’ve collected. Options include Metabase, Looker, and Hex.
```
```suggestion
1. A data warehouse to store all the data and act as a single source of truth. Options include Snowflake, BigQuery, and RedShift.

2. An ETL pipeline to extract data from different sources, transform it, and load it into the warehouse. Options include Fivetran, Integrate.io, and Airbyte.

3. A data transformation tool to model data, clean it up, and make it usable. dbt is basically the only option.

4. A business intelligence or visualization tool to get insights from the data you’ve collected. Options include Metabase, Looker, and Hex.
```

andyvan-ph replied:
Should we add reverse ETL at well... just to make it feel extra complicated? 🤔

ivanagas replied:
Data Orchestration and Data Governance are other options.

andyvan-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,108 @@
+4. A business intelligence or visualization tool to get insights from the data you’ve collected. Options include Metabase, Looker, and Hex.
+![Series A](../images/blog/startup-data-stack/series-a.png)
+
+Startups need this stack because managing and accessing individual sources of data becomes unsustainable. Tools like analytics and the CDP don't give complete insight into performance. Teams need a single source of truth, and the modern data stack gives them that. This creates trust in the data and greater accuracy.
```
```suggestion
Startups need this stack because managing and accessing individual sources of data becomes unsustainable. Tools like analytics and the CDP don't give complete insight into performance. Teams need a reliable, single source of truth they trust, and they hope the modern data stack will give them that.
```

andyvan-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,108 @@
+
+Startups need this stack because managing and accessing individual sources of data becomes unsustainable. Tools like analytics and the CDP don't give complete insight into performance. Teams need a single source of truth, and the modern data stack gives them that. This creates trust in the data and greater accuracy.
+
+The downside of this stage is that a gap opens between engineers and the data that is valuable to them. They are unable to "self-serve" and must learn the modern data stack tools or rely on the data team for insights. There are many reasons for this:
```
```suggestion
But this also where things start to go seriously wrong. All this complexity creates a gap between engineers and the data that is valuable to them. They are unable to "self-serve" and must learn the modern data stack tools, or rely on the data team for insights. There are many reasons for this:
```

andyvan-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,108 @@
+---
+title: "How the startup data stack evolves, what we’ve found"
```
```suggestion
title: "The "modern data stack" sucks"
```

andyvan-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,108 @@
+
+To do this, they need:
+
+- Website and product analytics
```
```suggestion
- Website and/or product analytics
```

andyvan-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,108 @@
+To do this, they need:
+
+- Website and product analytics
+- Customer relationship manager (CRM)
```
```suggestion
- A customer relationship manager (CRM)
```

andyvan-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,108 @@
+- Data security, safety, and privacy requirements.
+- Lack of knowledge of data available or how to use it.
+
+For example, engineers might need to write optimized SQL queries to understand customer feature usage. This requires an understanding of SQL, data availability, data structured, as well as access. These all create friction on what engineers want to do: build better products. 
```
```suggestion
For example, engineers might need to write optimized SQL queries to understand customer feature usage. This requires an understanding of SQL, data availability, data structured, as well as access. All this does is slow down their work, blocking them shipping new features and improvements efficiently.
```

andyvan-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,108 @@
+
+For example, engineers might need to write optimized SQL queries to understand customer feature usage. This requires an understanding of SQL, data availability, data structured, as well as access. These all create friction on what engineers want to do: build better products. 
+
+## PostHog’s vision for the future of the startup data stack
```
```suggestion
## How we want to fix the "modern data stack"
```

andyvan-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,108 @@
+
+## PostHog’s vision for the future of the startup data stack
+
+PostHog is not content with the "modern data stack." The gaps and complexity it creates limits the ability to [build successful products](/handbook/strategy/overview). To change this, we are improving how startups gather and analyze data at the earliest stages and consolidating more of the stack as they scale.
```
```suggestion
If you haven't figured it out already, we're not fans of this situation. To us, the "modern data stack" just creates costly barriers to building successful products. We want streamline how startups gather and analyze data at the earliest stages, consolidating more of the stack as they scale and delaying the need hire expensive heads of data and the like.
```

andyvan-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,108 @@
+
+PostHog is not content with the "modern data stack." The gaps and complexity it creates limits the ability to [build successful products](/handbook/strategy/overview). To change this, we are improving how startups gather and analyze data at the earliest stages and consolidating more of the stack as they scale.
+
+To make this happen, we are improving PostHog as a [customer data platform (CDP)](https://github.com/PostHog/posthog/issues/13126). We have built the key functionality, such as the ability to:
+
+- Receive data from your app(s) and site(s) with our [SDKs](/docs/integrate?tab=sdks)
+- Receive data from sources like [Stripe](/apps/stripe-connector), [Hubspot](/apps/hubspot-connector), [Intercom](/apps/intercom)
+- Export to destinations such as [Snowflake](/apps/snowflake-export), [BigQuery](/apps/bigquery-export), or [RedShift](/apps/redshift-export)
+
+Improving reliability, integrations, and UX is critical for success here. Being a CDP enables startups to have their data and tools in one place with PostHog. More data also improves the depth of insights engineers can get from PostHog without having to use other tools such as BI or visualization.
+
+![PostHog CDP](../images/blog/startup-data-stack/posthog-cdp.png)
+
+We are also figuring out what PostHog looks like as [a data warehouse](https://github.com/PostHog/posthog/issues/14406) utilizing [ClickHouse](/docs/how-posthog-works/clickhouse). Data accuracy and reporting flexibility matter at scale. Being a warehouse enables PostHog to provide this to startups, and integrate it with tools they are already using PostHog for. With PostHog as a warehouse, startu ...
+
+We want to make it easy to evaluate whether products are working or not. The ability for engineers to find and analyze the metrics they want, along with tools like experiments and session recordings, is key to this. The modern data stack creates complexity and silos that make figuring out what to work on and what’s working difficult.
+
+Integrating more of the parts of the startup data stack into PostHog enables engineers to continue to access the data, insights, and tools they need to build a great product. This current data stack evolution becomes large, complicated, and siloed quickly. Building an ideal data stack for engineers is what PostHog aims to do, and by doing so, we'll achieve our goal of more successful products in  ...
```
Maybe we can update this section to discuss what's been built at the offsite + HogQL. It feels a bit stronger now we've started building this stuff instead of just talking about it.

ivanagas replied:
I've rewritten this, didn't mention HogQL, but did talk about data warehousing (which is more relevant I think).

andyvan-ph reviewed (commented):
Been meaning to get back to this and give it a more opinionated tone. Have a look at the suggestions and let me know what you think. We've got a stronger story to tell now we've started to build some of this stuff.

ivanagas commented:
Reading through more modern data stack blogs, I love how they say "the modern data stack solves the problems of the traditional data stack by being more flexible and simpler." Then they say it requires 5 different types of tools, each with many options 🤔

ivanagas commented:
The important RFC for the last section: https://github.com/PostHog/product-internal/pull/458

andyvan-ph commented on contents/blog/startup-data-stack.md:
```diff
@@ -0,0 +1,136 @@
+
+devolves into this...
+
+![Series A](../images/blog/startup-data-stack/series-a.png)
```
Maybe we use your meme image here instead @ivanagas

andyvan-ph commented:
This is looking really good now. Just the one remaining comment from me, but otherwise it's good to go.

ivanagas commented:
Good to go now @andyvan-ph . Last question, should the title be:
1. The "modern data stack" sucks
or
2. The "startup data stack" sucks
Because we mostly talk about the "startup data stack," but "modern data stack is a more recognizable term.

andyvan-ph commented:
> Good to go now @andyvan-ph . Last question, should the title be:
> 
> 1. The "modern data stack" sucks
>    or
> 2. The "startup data stack" sucks
>    Because we mostly talk about the "startup data stack," but "modern data stack is a more recognizable term.

I think modern data stack. I did think about qualifying it and saying "The modern data stack sucks for startups", but the caveat feels a bit weak.

Thoughts @charlescook-ph?

charlescook-ph commented:
Let's say the modern data stack sucks - it's what we believe. Couple of other bits:
- I think we should always say modern data stack without air quotes, otherwise it sounds like we either a) are some corporation who are trying to write about the next cool thing (the so-called 'XBox'), or b) we're being unnecessarily sarcastic
- Images don't look good in dark mode
![Screenshot 2023-04-12 at 10 18 51](https://user-images.githubusercontent.com/70321811/231413651-51f25233-776b-4c9c-8fb1-91093dd759a7.png)
